addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -360,6 +361,7 @@ def _get_component_data(self, parent_bom, parent_product, warehouse, bom_line, l
             'quantity': line_quantity,
             'quantity_available': quantities_info.get('free_qty', 0),
             'quantity_on_hand': quantities_info.get('on_hand_qty', 0),
+            'free_to_manufacture_qty': quantities_info.get('free_to_manufacture_qty', 0),
```

Comment:
It's basically the same than `quantity_available`/`quantities_info.get('free_qty', 0),`
```suggestion
```